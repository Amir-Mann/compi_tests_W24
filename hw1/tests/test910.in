mXwÁ÷kÍðd
È²¼ùß0£-RíûÓ4ÊDJÒ$öA£2YÔ£2QÓðd*p½É| Òa\ImìªTì"~ßÃÁxÎfÏðÙ4§F×{