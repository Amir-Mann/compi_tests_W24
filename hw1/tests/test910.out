1 ID mX
Error 
